Makefile

```diff
@@ -115,6 +115,9 @@ reset-database: ##@database Resets the DB schema and runs migrations
 reset-fixtures: ##@database Resets the DB contents and reloads fixtures
 	docker-compose run --rm api sh scripts/reset_db_fixtures_local.sh
 
+db-terminal: ##@database Login to the database via the terminal
```

Comment:
I use the terminal to run SQL commands etc a fair bit rather than a GUI client so I've added this command to quickly log into the docker container and than log straight into the database :) As I'm lazy and don't fancy writing 2 commands every time 😆